src/app/notifications/toast/toast-sample-3/toast-sample-3.component.scss

```diff
@@ -1,5 +1,3 @@
-.div
-{
-    height: 100%; 
-    width: 100%;
-}
\ No newline at end of file
+button {
+    margin: 1% 1% 1% 0;
```

Comment:
I'd use pixel values here, just so that samples are more visually consistent across devices.